test/dotnet-core-uninstall.Tests/Shared/IntegrationTests/IntegrationTests.cs

```diff
@@ -1,21 +0,0 @@
-﻿using System.CommandLine;
-using System.CommandLine.IO;
-using System.CommandLine.Parsing;
-using FluentAssertions;
-using Microsoft.DotNet.Tools.Uninstall.Shared.Configs;
-using Xunit;
-
-namespace Microsoft.DotNet.Tools.Uninstall.Tests.Shared.IntegrationTests
-{
-    public class IntegrationTests
-    {
-        [Fact]
-        internal async void VersionCommandOutputsVersionInfo()
-        {
-            var console = new TestConsole();
-            var output = await CommandLineConfigs.UninstallCommandParser.InvokeAsync(new string[1] { "--version" }, console);
-            output.Should().Be(0);
-            console.Out.ToString().Should().Match("*.*.*");
-        }
-    }
-}
```

Comment:
I removed this test because it's not useful/relevant - the S.CL framework provides `--version` and we should just rely on it.